### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [kentrepreneur/prj-rev-bwfs-dasmoto#3]
When including images on your site it is recommended to add that [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) for each one. The reasons for this is because the attribute is used for both [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup, in case there is an error, and the image cannot be rendered. 

https://github.com/kentrepreneur/prj-rev-bwfs-dasmoto/blob/57f8a4fff5e5c94dcfd592d5349bc1450b9eb43e/Dasmoto%20Arts/index.html#L18
